spec/models/server_role_spec.rb

```diff
@@ -55,7 +55,7 @@
         web_services,Web Services,0,false,region
       CSV
 
-      allow(File).to receive(:open).and_return(StringIO.new(@csv))
+      allow(File).to receive(:open).with(ServerRole.fixture_path, "r", any_args).and_return(StringIO.new(@csv))
```

Comment:
This wasn't actually necessary, I just don't think mocking `File.open` without any args is error prone